packages/core/src/connectors/utilities/index.ts

```diff
@@ -1,22 +1,10 @@
 import { ConnectorConfig } from '@logto/schemas';
 
-import RequestError from '@/errors/RequestError';
 import { findConnectorById, updateConnector } from '@/queries/connector';
 
 export const getConnectorConfig = async <T extends ConnectorConfig>(id: string): Promise<T> => {
   const connector = await findConnectorById(id);
 
-  // FIXME:
-  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-  if (!connector) {
-    throw new RequestError({
-      code: 'entity.not_exists_with_id',
-      name: 'connector',
-      id,
-      status: 404,
-    });
-  }
-
   return connector.config as T;
```

Comment:
this looks weird, why do we need `as` here?

Reply:
because config is a generic type (empty object), but the user can pass in a more specific type T,

Reply:
let's discuss it tmw @wangsijie